### PR TITLE
Phone number formatter

### PR DESCRIPTION
**Pull request by sva605**

### CODE REVIEW

File: web/src/main/webapp/WEB-INF/views/registration.jsp [hunk 1]
```diff
@@ -16,6 +16,8 @@
     <link rel="stylesheet" href="css/bootstrap-theme.min.css">
     <link rel="stylesheet" href="css/slick.css">
     <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/sva.css">
```
Comment (backumich) [inline web/src/main/webapp/WEB-INF/views/registration.jsp:19]: sva? it doesn't describe what it contains

File: web/src/main/webapp/javascript/phoneFormat/intlTelInput.js [hunk 2]
```diff
@@ -0,0 +1,1156 @@
...
...["French Guiana (Guyane française)", "gf", "594"], ["French Polynesia (Polynésie française)", "pf", "689"], ["Gabon", "ga", "241"], ["Gambia", "gm", "220"], ["Georgia (საქართველო)", "ge", "995"], ["Germany (Deutschland)", "de", "49"], ["Ghana (Gaana)", "gh", "233"], ["Gibraltar", "gi", "350"], ["Greece (Ελλάδα)", "gr", "30"], ["Greenland (Kalaallit Nunaat)", "gl", "299"], ["Grenada", "gd", "1473"], ["Guadeloupe", "gp", "590", 0], ["Guam", "gu", "1671"], ["Guatemala", "gt", "502"], ["Guernsey", "gg", "44", 1], ["Guinea (Guinée)", "gn", "224"], ["Guinea-Bissau (Guiné Bissau)", "gw", "245"], ["Guyana", "gy", "592"], ["Haiti", "ht", "509"], ["Honduras", "hn", "504"], ["Hong Kong (香港)", "hk", "852"], ["Hungary (Magyarország)", "hu", "36"], ["Iceland (Ísland)", "is", "354"], ["India (भारत)", "in", "91"], ["Indonesia", "id", "62"], ["Iran (‫ایران‬‎)", "ir", "98"], ["Iraq (‫العراق‬‎)", "iq", "964"], ["Ireland", "ie", "353"], ["Isle of Man", "im", "44", 2], ["Israel (‫ישראל‬‎)", "il", "972"], ["Italy (Italia)", "it", "39", 0], ["Jamaica", "jm", "1876"], ["Japan (日本)", "jp", "81"], ["Jersey", "je", "44", 3], ["Jordan (‫الأردن‬‎)", "jo", "962"], ["Kazakhstan (Казахстан)", "kz", "7", 1], ["Ken
```
Comment (backumich) [inline web/src/main/webapp/javascript/phoneFormat/intlTelInput.js]: did you copy that from google libs?

Comment (backumich) [inline web/src/main/webapp/javascript/phoneFormat/intlTelInput.js]: I bet you've copied all of that

File: web/src/main/webapp/javascript/phoneFormat/utils.js [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+,,,[[,"(\\d)(\\d{3})(\\d{3})","$1 $2 $3",,"0$1"]],,[,,,,,,,,,[-1]],,,[,,,,,,,,,[-1]],[,,,,,,,,,[-1]],,,[,,,,,,,,,[-1]]],ES:[,[,,"[5-9]\\d{8}",,,,,,,[9]],[,,"8(?:[1356]\\d|[28][0-8]|[47][1-9])\\d{6}|9(?:[135]\\d{7}|[28][0-8]\\d{6}|4[1-9]\\d{6}|6(?:[0-8]\\d{6}|9(?:0(?:[0-57-9]\\d{4}|6(?:0[0-8]|1[1-9]|[2-9]\\d)\\d{2})|[1-9]\\d{5}))|7(?:[124-9]\\d{2}|3(?:[0-8]\\d|9[1-9]))\\d{4})",,,,"810123456"],[,,"(?:6\\d{6}|7[1-48]\\d{5}|9(?:6906(?:09|10)|7390\\d{2}))\\d{2}",,,,"612345678"],[,,"[89]00\\d{6}",,,,"800123456"],
+[,,"80[367]\\d{6}",,,,"803123456"],[,,"90[12]\\d{6}",,,,"901123456"],[,,"70\\d{7}",,,,"701234567"],[,,,,,,,,,[-1]],"ES",34,"00",,,,,,,,[[,"([89]00)(\\d{3})(\\d{3})","$1 $2 $3",["[89]00"]],[,"([5-9]\\d{2})(\\d{2})(\\d{2})(\\d{2})","$1 $2 $3 $4",["[568]|[79][0-8]"]]],,[,,,,,,,,,[-1]],,,[,,,,,,,,,[-1]],[,,"51\\d{7}",,,,"511234567"],,,[,,,,,,,,,[-1]]],ET:[,[,,"[1-59]\\d{8}",,,,,,,[9],[7]],[,,"(?:11(?:1(?:1[124]|2[2-57]|3[1-5]|5[5-8]|8[6-8])|2(?:13|3[6-8]|5[89]|7[05-9]|8[2-6])|3(?:2[01]|3[0-289]|4[1289]
```
Comment (backumich) [inline web/src/main/webapp/javascript/phoneFormat/utils.js]: and that

**Review by backumich — CHANGES_REQUESTED** [review]
no copy-paste, only import allowed

File: web/src/main/webapp/WEB-INF/views/registration.jsp (hunk 1 above)
Comment (sva605) [reply web/src/main/webapp/WEB-INF/views/registration.jsp:19]: Changed

File: web/src/main/webapp/javascript/phoneFormat/intlTelInput.js (hunk 2 above)
Comment (sva605) [reply web/src/main/webapp/javascript/phoneFormat/intlTelInput.js]: No, from here: https://github.com/jackocnr/intl-tel-input

Comment (sva605) [reply web/src/main/webapp/javascript/phoneFormat/intlTelInput.js]: This is a ready-to-use code

File: web/src/main/webapp/javascript/phoneFormat/utils.js (hunk 3 above)
Comment (sva605) [reply web/src/main/webapp/javascript/phoneFormat/utils.js]: see above

**Comment by sva605** [comment]
copy-past removed

**Pull request merged**